README.md

```diff
@@ -1 +1,64 @@
-# To-Do-List
```

Comment:
- [Optional] Kindly add the License section of this template in [Your README](https://github.com/microverseinc/readme-template) file. This will render your README a professional one.